evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -108,6 +108,16 @@ def percentage_one_decimal(fraction, population):
         return None
 
 
+@register.filter
+def percentage_zero_on_error(fraction, population):
+    try:
+        return f"{int(float(fraction) / float(population) * 100):.0f}%"
+    except ValueError:
+        return "0%"
+    except ZeroDivisionError:
+        return "0%"
```

Comment:
we could combine the except clauses
```suggestion
    except (ZeroDivisionError, ValueError):
        return "0%"
```